### ISSUE TITLE
Exposition in A1

### ISSUE THREAD

**Issue by eenblam** [nborwankar/LearnDataScience#17]
> What the software did was compute a function called a "loss function", a measure of error. Then, it "tried out" multiple straight lines until it found one that minimized the "loss function" value for that choice -- then it read off the Intercept and X-slope for that line.

If you're doing linear regression, why would the software consider multiple lines? Isn't the point of performing linear regression that we can specify a loss function and easily solve for the specific coefficients which minimize it?

**Comment by nborwankar** [comment]
Sorry for the huge delay in response.  The informal point I was making is that it is "as if" the software was trying out multiple lines behind the scene.  What needs clarifying - I agree - is that it is not what happens when solving for coefficients.  What you are doing when solving for coefficients is minimizing the loss function and I am assuming "gradient descent" in my mental model.  So I agree it needs clarification - probably in the next big update.
